lib/ansible/executor/task_executor.py

```diff
@@ -268,7 +268,7 @@ def _run_loop(self, items):
                     u" to something else to avoid variable collisions and unexpected behavior." % loop_var)
 
         ran_once = False
-        items = self._squash_items(items, loop_var, task_vars)
+        items = self._squash_items(items, loop_var, task_vars, templar)
```

Comment:
_squash_items() doesnt take a templar arg

```
The full traceback is:
Traceback (most recent call last):
  File "/home/adrian/src/ansible/lib/ansible/executor/task_executor.py", line 99, in run
    item_results = self._run_loop(items, templar)
  File "/home/adrian/src/ansible/lib/ansible/executor/task_executor.py", line 271, in _run_loop
    items = self._squash_items(items, loop_var, task_vars, templar)
TypeError: _squash_items() takes exactly 4 arguments (5 given)

fatal: [localhost]: FAILED! => {
    "failed": true, 
    "msg": "Unexpected failure during module execution.", 
    "stdout": ""
}
```

Adding a templar arg gets it pass the error, but the memory usage doesnt seem to improved